server/build/webpack.js

```diff
@@ -178,6 +189,7 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
       ],
       loaders
     },
+    devtool: dev ? 'inline-source-map' : false,
```

Comment:
@jonaswindey I'm merging, but if there's a faster alternative that works, please let us know